configs/nri-prometheus-config.yml.sample

```diff
@@ -32,6 +32,14 @@ integrations:
       # Length in time to distribute the scraping from the endpoints.
       scrape_duration: "5s"
 
+      # Number of worker threads used for scraping targets.
+      # For large clusters with many (>100) endpoints, slowly increase until scrape
```

Comment:
In the documentation we specify 400 as a safe value, we should be consistent

Reply:
https://docs.newrelic.com/docs/integrations/prometheus-integrations/install-configure-openmetrics/configure-prometheus-openmetrics-integrations-large-kubernetes-environments